src/week1/WaterMotorAutomation.java

```diff
@@ -0,0 +1,5 @@
+package week1;
+
+public interface WaterMotorAutomation {
+    public void automateWaterMotor(double level);
```

Comment:
method should be automate. interface name is clear to identify that water motor is being automated